libs/blocks/quiz/quiz.js

```diff
@@ -57,45 +60,100 @@ const App = () => {
       const currentflow = userFlow.shift();
```

Comment:
should be currentFlow and should we have comments for this function too?

Reply:
I have not added comments for all the functions knowingly. Just for the ones which might be a little heavy on logic and Wiki knowledge. Will change the variable names.